src/config/tolgeerc.ts

```diff
@@ -76,7 +76,7 @@ function parseConfig(rc: any): TolgeeConfig {
       throw new Error('Invalid config: delimiter is not a string');
     }
 
-    cfg.delimiter = rc.delimiter || void 0;
+    cfg.delimiter = rc.delimiter || '';
```

Comment:
Doesn't this make '' the default? As the default should be ".".

Reply:
It shouldn't and people expect this behavior when they explicitly set the delimiter to null (or an empty string) as it carries the meaning of "no delimiter". The default is applied if the delimiter is unspecified in the config (in this case it'll be "."). See #55

Reply:
👍